### ISSUE TITLE
PHP Notice:  Undefined offset: 1 in class-wc-ebanx-environment.php on line 71

### ISSUE THREAD

**Issue by romuloctba** [ebanx/woocommerce-gateway-ebanx#689]
I believe this is due the use of filter_input for INPUT_SERVER that fails on fastcgi enviroments

So when you do `filter_input( INPUT_SERVER, 'SERVER_SOFTWARE' )` at the class WC_EBANX_Environment, you might not get the desired information if the server runs fastcgi, and thus would no be able to rely in the exploded array as a source of info

This problem is known for a while now, and there are some workarounds
http://php.net/manual/en/function.filter-input.php#77307

According to the comment above, one can use the filter_input for cookies, get and post, which are usually the real needs of it. But for server information, you might want to go directly with $_SERVER and $_ENV
Relevant code at WC_EBANX_Environment

```
$web_server_information_string = filter_input( INPUT_SERVER, 'SERVER_SOFTWARE' );
$web_server_value_parts_array  = explode( ' ', $web_server_information_string );
$web_server_parts = explode( '/', $web_server_value_parts_array[0] );
$web_server = new stdClass();
$web_server->name  = str_replace( '-', ' ', $web_server_parts[0] );
$web_server->version = $web_server_parts[1];
```

Still, the webserver version could be disable, thus it would still not fetchable, but if you go with the $_SERVER instead of the INPUT_SERVER + several eploding , we would not see the `Undefined offset: 1 ` notice

**Comment by romuloctba** [comment]
Proposed fix on #692

**Comment by andercampanha** [comment]
PR with the fix has just been merged. Thank you.
